### PR TITLE
chore: Update Build Scripts

### PR DESCRIPTION
**Pull request by alexrisch**
Added App Release Action
Removed schedule on internal action
Increment app version for next build

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "expo": {
-    "version": "2.0.3",
+    "version": "2.0.4",
     "ios": {
-      "buildNumber": "4"
+      "buildNumber": "1"
```
Comment (thierryskoda) [inline app.json:5]: I think Apple might complain because they always want a buildNumber higher than what was previously released 😮

**Review by thierryskoda — APPROVED** [review]

File: app.json (hunk 1 above)
Comment (thierryskoda) [reply app.json:5]: I love using auto increment for that https://docs.expo.dev/build-reference/app-versions/#remote-version-source

Comment (thierryskoda) [reply app.json:5]: But maybe we can add this later

Comment (alexrisch) [reply app.json:5]: buildNumber needs to be higher based on the appVersion, so it should always increase the app version first, and then reset this to "1"

Comment (Koleok) [reply app.json:5]: yeah actually with that option set, you may be able to get rid of the extra line in the version bump script.

**Pull request merged**